run/idp-sql/test/system.test.js

```diff
@@ -0,0 +1,115 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const admin = require('firebase-admin');
+const assert = require('assert');
+const path = require('path');
+const supertest = require('supertest');
+const got = require('got');
+const uuid = require('short-uuid');
+const {execSync} = require('child_process');
+
+describe('System Tests', () => {
```

Comment:
Now/Future Enhancement: Given all the setup overhead for end-to-end tests, deeper coverage is worthwhile. It will cost seconds and we're already invested a couple minutes.

The existing "light" test coverage is not because it's preferred but because that's a level of test coverage I could add to many of the samples in an afternoon.

Reply:
I'm not sure how to extend test coverage here. Since we deploy with authentication, that conflicts with the authentication of POST request route.

Reply:
We can special case this to deploy without authentication.

The principle is that apps are not publicly accessible, but what you're saying is that this (mostly) isn't, because of a different auth mechanism.